### DISCUSSION TITLE
Add support for Odysee videos

### DISCUSSION THREAD

**Discussion by darwin-brandao** [FreeTubeApp/FreeTube#4705] [General]
First of all, FreeTube is absolutely amazing and I use it daily. Thanks A LOT for this great YouTube client, I practically don't use YouTube to watch videos anymore.

I know that the project's objective is to be a privacy-oriented YouTube client, and I think it would be great if the devs added support for other platforms, privacy-oriented ones, such as Odysee.

I know it is out of the original scope of the project and it might get a little too complex to maintain, but I couldn't keep me from proposing it, because I also love Odysee content (mostly the privacy related ones) and it would be GREAT to have both Odysee and YouTube content on FreeTube.

**Comment by absidue** [comment]
https://docs.freetubeapp.io/faq/#will-you-consider-adding-support-for-services-outside-of-youtube-twitch-soundcloud-peertube-etc
